### ISSUE TITLE
Make an overview of the various bug-trackers

### ISSUE THREAD

**Issue by MichaelMure** [git-bug/git-bug#30]
When designing git-bug it would be very useful to have an overview of the various bug-trackers that are out there.  Here is a sample of questions that I have:

## Actor

- what properties (name, email, login, id, avatar ...) does the bug-tracker know about an actor ?
- what is mandatory, what is optional ?

## Bug's state

- how do they model a state ? Is it an fixed set of state, or is it extensible? configurable ?
- how do they model the transition between these states ?

## missing features in git-bug

- For instance, Github support reactions to messages with a set of emojis. What other bug-tracker support that ?
- What other important features other bug-trackers have ? Is there a common ground ? Are they incompatible ?

With the attention git-bug got lately, hopefully we can collaborate and fill the blanks together in a shared spreadsheet.

**Comment by j-f1** [comment]
Things in *italics* are missing features.

# GitHub

Actor: name (not shown by default), *username/login*, email (not shown by default, but may be useful for associating the user with commits), *avatar*
State: Open/closed, *locked/unlocked*, *optional lock reason*, *assignees* (≤ 10)
Categorization: labels (name + description + *color*), *milestone* (1 per issue), *projects* (multiple, can place issues in different columns)

**Comment by MichaelMure** [comment]
@j-f1 thank you !

I started a shared spreadsheet, could move over there ? https://framacalc.org/bugtracker

Sorry you are way too fast ;)

**Comment by MichaelMure** [comment]
Also we should stay with facts only for now imho, we will decide from that what feature we can and should support, and how.

**Comment by neithernut** [comment]
Maybe also have a look at https://dist-bugs.branchable.com/software/ (in theory, this link should have reached you already via various channels)

**Comment by MichaelMure** [comment]
@neithernut yes thanks I saw it. We need the extracted data version to make educated decision on the design.

**Comment by MichaelMure** [comment]
@neithernut maybe join the gitter channel ? https://gitter.im/the-git-bug/Lobby

**Comment by MichaelMure** [comment]
Feel free to add new features/item as you see fit.

**Comment by MichaelMure** [comment]
[bugtracker.xlsx](https://github.com/MichaelMure/git-bug/files/2720982/bugtracker.xlsx)

Backup in case the online version get lost.
